src/probes/isRequire.js

```diff
@@ -163,8 +183,5 @@ function walkRequireCallExpression(nodeToWalk, tracer) {
 
 export default {
   name: "isRequire",
-  validateNode,
-  main,
-  breakOnMatch: true,
-  breakGroup: "import"
+  validateNode: [validateNodeRequire, validateNodeEvalRequire], main, breakOnMatch: true, breakGroup: "import"
```

Comment:
you can keep break lines for each properties

Reply:
agree its fixed in latest commit